crates/sparrow-runtime/src/metadata/raw_metadata.rs

```diff
@@ -13,10 +13,10 @@ use sparrow_api::kaskada::v1alpha::source_data::{self, Source};
 use sparrow_api::kaskada::v1alpha::PulsarConfig;
 use tracing::info;
 
-use crate::execute::pulsar_schema;
 use crate::metadata::file_from_path;
 use crate::stores::object_store_url::ObjectStoreKey;
 use crate::stores::{ObjectStoreRegistry, ObjectStoreUrl};
+use crate::streams::pulsar_schema;
```

Comment:
I wonder if we should have a pattern here of exposing a module for each "kind of stream", so this would be `crate::streams::pulsar::schema`?

Reply:
Yeah, I figured we'd move to that pattern once we get another stream. I can go ahead and do that now